tests/test_photonphase.py

```diff
@@ -36,5 +37,40 @@ def test_result():
     photonphase.sys.stdout = saved_stdout
 
 
+parfile_nicer = os.path.join(datadir, "ngc300nicer.par")
+parfile_nicerbad = os.path.join(datadir, "ngc300nicernoTZR.par")
+eventfile_nicer = os.path.join(datadir, "ngc300nicer_bary.evt")
+
+
+@pytest.mark.skipif(
+    "DISPLAY" not in os.environ, reason="Needs an X server, xvfb counts"
+)
+def test_nicer_result():
+    "Check that barycentered NICER data is processed correctly."
+    saved_stdout, photonphase.sys.stdout = photonphase.sys.stdout, StringIO("_")
+    cmd = "{0} {1}".format(eventfile_nicer, parfile_nicer)
+    photonphase.main(cmd.split())
+    lines = photonphase.sys.stdout.getvalue()
+    v = 0.0
+    for l in lines.split("\n"):
+        if l.startswith("Htest"):
+            v = float(l.split()[2])
+    # Check that H-test is greater than 725
+    log.warning("V:\t%f" % v)
```

Comment:
No need for this, pytest will tell us the value of v if it is smaller.

Reply:
Yeah, I don't know why that is there. Must have come from initial debugging of the tests